src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  li {
+    background: url(../images/star.svg) var(--background-properties);
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+  }
+
+  :nth-child(-n + 4) {
+    background: url(../images/star-active.svg) var(--background-properties);
```

Comment:
You always have 4 active stars, but functionality should be dynamic. If the block has modifier `stars--3` you should show 3 active stars, and so on